flixel/math/FlxPoint.hx

```diff
@@ -152,6 +152,19 @@ class FlxPoint implements IFlxPooled
 	}
 	
 	/**
+	 * Multiply this point by a scalar.
+	 * 
+	 * @param	scalar	How much times this point will be as it before.
+	 * @return	This point.
+	 */
+	public function multiply(scalar:Float):FlxPoint
```

Comment:
It is not redundant.
Some of HaxeFlixel's code use FlxPoint instead of FlxVector.
So you have to write code like this:
```
point.x*=scalar;
point.y*=scalar;
```